tests/components/tts/test_init.py

```diff
@@ -176,6 +176,41 @@ async def test_setup_component_and_test_service_with_config_language(
     ).is_file()
 
 
+async def test_setup_component_and_test_service_with_config_language_special(
+    hass, empty_cache_dir
+):
+    """Set up the demo platform and call service with extend language."""
+    import homeassistant.components.demo.tts as demo_tts
+
+    demo_tts.SUPPORT_LANGUAGES.append("en_US")
```

Comment:
Won't this change to the constant remain after this test is run?

Reply:
Maybe, however, it have only a usage if you load it, otherwise it's just around

Reply:
It's bad practice to have persistent effects in tests as it might effect other (future) tests unexpectedly.